snuba/web/views.py

```diff
@@ -179,25 +195,25 @@ def handle_invalid_query(exception: InvalidQueryException) -> Response:
 
 
 @application.route("/")
-def root():
+def root() -> str:
     with open("README.md") as f:
         return render_template("index.html", body=markdown(f.read()))
 
 
 @application.route("/css/<path:path>")
-def send_css(path):
+def send_css(path: str) -> Any:
     return application.send_static_file(os.path.join("css", path))
 
 
 @application.route("/img/<path:path>")
 @application.route("/snuba/web/static/img/<path:path>")
-def send_img(path):
+def send_img(path: str) -> Any:
     return application.send_static_file(os.path.join("img", path))
 
 
 @application.route("/dashboard")
 @application.route("/dashboard.<fmt>")
-def dashboard(fmt="html"):
+def dashboard(fmt: str = "html") -> Any:
```

Comment:
`application.send_static_file` resolves to Any and not Response.

Reply:
Really? The Mypy revealed type reports this to be Response to me

Reply:
fixed